test-data/unit/check-flags.test

```diff
@@ -515,3 +515,95 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyDecoratedUnannotatedDecorator]
```

Comment:
Some additional tests I'd like to see:
- test that unannotated functions and functions annotated with `Any` that are undecorated do not produce an error
- test the behavior on a function that is has two decorators -- one which makes it untyped and then one which makes it typed (without Anys) again